src/content/docs/en/pages/guides/marketplace/templates/bot-with-tor.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: How to block Tor Exit Nodes requests using a template
+description: Deploy a basic security suite to protect your domains from bad bots and Tor Exit Nodes requests.
+meta_tags: templates, guides, Azion Marketplace, security, edge firewall
+namespace: docs_guides_templates_bot_manager_tor_list
+permalink: /documentation/products/guides/block-tor-exit-nodes-template/
+---
+
+import Badge from '~/components/Badge.astro';
+
+<Badge variant="accent">
+Preview
+</Badge>
+
+The **Bot Manager and TOR Exit Nodes** template enables you to deploy a basic security suite to protect your domains from bad bots and Tor Exit Nodes requests.
+
+This template automatically creates a new **Edge Firewall** instance that includes the rules to execute an edge function instance, confirm if your domains are valid, and deny requests from the pre-built "Azion IP Tor Exit Nodes" network list. All of this, working in conjunction with the **Edge Functions** and **Network Layer Protection** modules and **Bot Manager**.
+
+---
+
+## Requirements
+
+Before deploying this template, you must:
+
+- Have the **Edge Functions** and **Network Layer Protection** modules enabled in your account.
+    - To do so, go to the [Billing & Subscriptions](https://manager.azion.com/billing-subscriptions/subscriptions?active_tab=subscriptions) section and activate the switch for the module.
+    - If these modules are activated, executing this template could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
+- Launch the latest version of the **Bot Manager** integration.
+    - Go to [Azion Marketplace](https://console.azion.com/marketplace) to install this integration by clicking the **Install* button. Check the [How to install Azion Bot Manager](/en/documentation/products/guides/bot-manager/) guide for more details.
+    - If you didn't install the Bot Manager function before deploying this template, the execution will fail and a log explaining the reason will be printed.
+
+---
+
+## Getting the template
+
+To use this template, access [Azion Console](https://console.azion.com/) and select the **Start with a template** option on the homepage.
+
+This will open a modal where you can select the **Templates** > **Bot Manager and TOR Exit Nodes** card.
```

Comment:
```suggestion
To use this template, access [Azion Console](https://console.azion.com/) and click the **+ Create** button on the homepage.

This will open a modal where you can select the **Templates** > **Bot Manager and TOR Exit Nodes** card.
```